### ISSUE TITLE
AMT problem after 30+- days of use

### ISSUE THREAD

**Issue by trenton92** [Ylianst/MeshCentral#1602]
Hey.
Sorry that is off topic.
I saw on reddit they wrote on reddit about this problem.
https://www.reddit.com/r/MeshCentral/comments/fddflj/intelamtconnect_authentication_error/

I had this using CIRA.
I thought the problem was in CIRA and started testing on the local network.
The problem remains, about once a month this problem pops up.
Helps to change and return settings in MEBx settings.

Perhaps someone faced such a problem and could solve it.
I will be very grateful for your help.

**Comment by Ylianst** [comment]
I still don't have a proper description of the problem. I will close this issue. If you can open again describing exactly what the problem is with as much detail as possible, that would be great. Thanks.

**Comment by trenton92** [comment]
Sorry for the poor description of the problem.
I activate Intel AMT. I use it for about a month in MeshCommander. After this period, an authentication error is thrown. I am trying to access localhost: 16992 through a browser. The browser also gives an authentication error.
I enter the correct name and password exactly. We use the Lenovo M900 mini-PC. I configured AMT in one day to all. And after about a month, absolutely all mini-pc connected to the network give an authentication error.
I know that the problem is not with MeshCommander. I did not find a solution to the problem on the Internet.
I just saw that other users of MeshCentral had this problem and wanted to know if someone had solved them.
